python/pyspark/sql/functions/builtin.py

```diff
@@ -14533,7 +14533,7 @@ def arrays_overlap(a1: "ColumnOrName", a2: "ColumnOrName") -> Column:
 
 @_try_remote_functions
 def slice(
-    x: "ColumnOrName", start: Union["ColumnOrName", int], length: Union["ColumnOrName", int]
```

Comment:
it will be a breaking change for `slice(x=..., start=...)`

since we have already documented the parameters `x`, I think we should not change hare, instead we can change the connect side.

Reply:
Sounds reasonable to me. Just applied the comment. Thanks!